sdk/azurestackhci/Microsoft.Azure.Management.AzureStackHCI/src/Generated/ClustersOperationsExtensions.cs

```diff
@@ -183,12 +181,12 @@ public static async Task<Cluster> CreateAsync(this IClustersOperations operation
             /// <param name='clusterName'>
             /// The name of the cluster.
             /// </param>
-            /// <param name='tags'>
-            /// Resource tags.
+            /// <param name='cluster'>
+            /// Details of the HCI cluster.
             /// </param>
-            public static Cluster Update(this IClustersOperations operations, string resourceGroupName, string clusterName, IDictionary<string, string> tags = default(IDictionary<string, string>))
+            public static Cluster Update(this IClustersOperations operations, string resourceGroupName, string clusterName, ClusterPatch cluster)
```

Comment:
similr  method signatures needed here - again, a partial class in customizations will fix this.